ledger/accountdb_test.go

```diff
@@ -59,8 +60,11 @@ func checkAccounts(t *testing.T, tx *sql.Tx, rnd basics.Round, accts map[basics.
 
 	for addr, data := range accts {
 		pad, err := aq.lookup(addr)
-		d := pad.accountData
 		require.NoError(t, err)
+		d := pad.accountData
+		if !reflect.DeepEqual(d, data) {
+			fmt.Println("not equal", addr.String())
```

Comment:
what's the added value of this compared to the `require.Equal` statement below ?